internal/pwru/utils.go

```diff
@@ -26,8 +26,9 @@ func GetFuncs(pattern string) (Funcs, error) {
 	}
 	callback := func(typ btf.Type) {
 		fn := typ.(*btf.Func)
+		fnName := string(fn.Name)
 
-		if pattern != "" && !reg.Match([]byte(fn.Name)) {
+		if pattern != "" && reg.FindString(fnName) != fnName {
```

Comment:
`FindString` does a prefix match. Is that intended? With that code, `--filter-func=kfree_sbk would still match kfree_skbmem` https://go.dev/play/p/M3yIKZpCVf8